core/extension/gdextension_interface.h

```diff
@@ -3111,6 +3114,20 @@ typedef void (*GDExtensionsInterfaceEditorHelpLoadXmlFromUtf8Chars)(const char *
  */
 typedef void (*GDExtensionsInterfaceEditorHelpLoadXmlFromUtf8CharsAndLen)(const char *p_data, GDExtensionInt p_size);
 
+/**
+ * @name editor_register_get_classes_used_callback
+ * @since 4.5
+ *
+ * Registers a callback that Godot can call to get the list of all classes (from ClassDB) that may be used by the calling GDExtension.
+ *
+ * This is used for generating build profiles, in order to recompile Godot with only the classes used.
+ * If a GDExtension doesn't register a callback, Godot will assume that it could be using any classes.
```

Comment:
One possible suggestion:
```suggestion
 * This is used for generating build profiles, in order to recompile Godot with only the classes used.
 * A possible use case in the editor is "Tools" > "Engine Compilation Configuration Editor..." > "Detect from project."
 *
 * The GDExtension should provide the list of classes that _may_ be used statically, thus the time of invocation shouldn't matter.
 * If a GDExtension doesn't register a callback, Godot will assume that it could be using any classes.
```

Reply:
Thanks! I've integrated your suggest with some minor changes